src/main/java/frc/robot/Limelight.java

```diff
@@ -13,6 +15,25 @@ public boolean hasDetectedTarget(){
         return LimelightHelpers.getTV(name);
     }
 
+    public double getDistanceToTarget(){
+        LimelightHelpers.RawFiducial[] fiducials = LimelightHelpers.getRawFiducials(name);
+        if(fiducials.length >0)
+            return fiducials[0].distToRobot;
+        return 3;
```

Comment:
why 3? that is a lie, but it may fool someone into thinking the real distance is 3, return a number indicating an invalid value.

Either a negative value, optional, or if you want your trick to still work, return `Double.MAX_VALUE`